apps/wallet-dashboard/components/Dialogs/settings/enums/view.enums.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+export enum SettingsDialogView {
+    SelectSetting = 'SelectSetting',
+    NetworkSettings = 'NetworkSettings',
+    AutoLockProfile = 'AutoLockProfile',
+    FAQ = 'FAQ',
+    Themes = 'Themes',
+    Logout = 'Logout',
```

Comment:
we're just using `NetworkSettings` right?
I don't know if the rest are necessary, for example, we have the theme change outside the settings

Reply:
For now we can have here just `SelectSetting` & `NetworkSettings` indeed